codecov.yml

```diff
@@ -0,0 +1,10 @@
+codecov:
```

Comment:
can you link https://docs.codecov.com/docs/codecovyml-reference here?

Reply:
Done